tests/conftest.py

```diff
@@ -162,6 +160,16 @@ def __init__(
                 model_name,
                 torch_dtype=torch_dtype,
             )
+        elif model_name in _EMBEDDING_MODELS:
```

Comment:
@CatherineSue here is where you are running OOM

I pushed a fix for this to your branch, but looks like it got overridden

The model is getting loaded at fp32 here, so its consuming 7*4=28GB or RAM on a machine with 24GB of RAM

Load the model at FP16 here and you will be good to go

Reply:
thanks!! applied. Running CI now.

Reply:
Seems the test is still failing. 

>model-00001-of-00002.safetensors: 100% 9.94G/9.94G [00:55<00:00, 180MB/s]
  model-00002-of-00002.safetensors: 100% 4.28G/4.28G [00:23<00:00, 183MB/s]

I have changed the dtype to fp16.

Reply:
Passed device=cpu in SentenceTransformer init. This test is passed in Models Test now.